add-pkg.js

```diff
@@ -0,0 +1,59 @@
+const { execSync } = require("child_process");
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+
+
+async function run () {
+  const namespace = "@gwyneplaine";
+
+  const pkgName = await inquirer.prompt([{
+    message: "First, what is the name of your new package?",
+    name: 'pkgName',
+    validate: input => input.length > 0
+  }]).then(responses => {
+    return responses.pkgName
+  });
+
+  const entryPoint = await inquirer.prompt([{
+    message: "Next, please the entrypoint for your package? (i.e. index.js)",
+    name: 'entryPoint',
+    validate: input => input.length > 0
+  }]).then(responses => responses.entryPoint);
+
+  const packageConfig = await inquirer.prompt([{
+    message: 'Please specify your package manager (npm is the default)',
+    name: 'packageManager',
+  }]).then(({ packageManager }) => {
+    switch (packageManager.toLowerCase()) {
+      case 'npm':
```

Comment:
Is the plan to add cocoapods stuff later? Maybe leave a todo comment

Reply:
yep, I'll make it into an issue as well